test/bin/scenario.sh

```diff
@@ -1189,6 +1202,13 @@ action_run() {
 
     check_dependencies
 
+    for hname in "$@"; do
+	    if [[ ! "$hname" =~ ^host[0-9]+$ ]]; then
+	       RUN_HOST_OVERRIDE=( "$@" )
+	       break
+	    fi
+    done
```

Comment:
I'm not sure I understand why we need this check. This function is getting argv from the command line, so it's either empty or contains a single host or IP.

Reply:
Can't we just check if $1 exists and not empty and assign it?

Reply:
one question, somewhere in either action_run or run_tests i need to check if  the new variable matches host1 or not right ? if not host1, i will need to run set_vm_property to populate the variables.yaml file right ?

Reply:
I'm not sure host name validation is necessary.
What we need to do is to override the location where **real** VM host IP is derived. 

Could it be that it's sufficient to do this in the get_vm_ip function?